main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -2200,7 +2223,7 @@ public static boolean clearSearchedDestinations() {
 
         final Cursor cursor = database.query(
                 dbTableTrackables,
-                new String[]{"_id", "updated", "tbcode", "guid", "title", "owner", "released", "goal", "description"},
+                new String[]{"_id", "updated", "tbcode", "guid", "title", "owner", "released", "goal", "description", "retrieved"},
```

Comment:
Shouldn't you add the colums `disposition`, `disposition_type` and `disposition_log_guid`?

Reply:
Yep, thanks.